chart/templates/replicated-supportbundle.yaml

```diff
@@ -44,6 +44,11 @@ stringData:
             name: replicated-instance-report
             includeValue: true
             key: report
+        - secret:
+            namespace: {{ include "replicated.namespace" . }}
+            name: replicated-custom-app-metrics-report
+            includeValue: true
+            key: report
```

Comment:
good catch. could also add some validation here too if you want https://github.com/replicatedhq/replicated-sdk/blob/main/.github/workflows/main.yaml#L496